src/db/QDjangoQuerySet_p.h

```diff
@@ -95,11 +95,15 @@ class QDJANGO_DB_EXPORT QDjangoQuerySetPrivate
 
     // SQL queries
     QDjangoQuery countQuery() const;
+    QDjangoQuery aggregateQuery(const QString &field, QDjangoWhere::AggregateType func) const;
     QDjangoQuery deleteQuery() const;
     QDjangoQuery insertQuery(const QVariantMap &fields) const;
     QDjangoQuery selectQuery() const;
     QDjangoQuery updateQuery(const QVariantMap &fields) const;
 
+    //static aggregate functions mapping
+    static QString aggregationToString(QDjangoWhere::AggregateType type);
```

Comment:
I am not convinced we even need a method in the class, a static function in QDjangoQueryset.cpp should be sufficient